src/Models/Product.php

```diff
@@ -156,6 +157,24 @@ public function parent(){
             return $this->belongsTo(Product::class, 'parent_id');
         }
 
+        /**
+         * Tag relation
+         *
+         * @var $query
+         */
+        public function tags(){
+            return $this->morphToMany(Tag::class, 'taggable')->withTimestamps();
+        }
+
+        /**
+         * Tag relation
+         *
+         * @var $query
+         */
+        public function tag($tag){
```

Comment:
@AlexeiaW should we consider some kind of verb for this so it's clearer for the developer. Not sure what, maybe `setTag` or `addTag`, what do you think? Mainly to avoid confusion whereby the user might think that ->tag returns a single tag attached to the product.

Reply:
addTag feels more descriptive of what the function does, so renamed to that